bookkeeper-common/src/test/java/org/apache/bookkeeper/test/TestStatsProvider.java

```diff
@@ -60,10 +60,16 @@ public void dec() {
         }
 
         @Override
-        public void add(long delta) {
+        public void addCount(long delta) {
             updateMax(val.addAndGet(delta));
         }
 
+        @Override
+        public void addLatency(long eventLatency, TimeUnit unit) {
+            long valueMillis = unit.toMicros(eventLatency) / 1000;
```

Comment:
Just have a question. Why not use unit.toMills?

Reply:
> Just have a question. Why not use unit.toMills?

it's the same, just use the logic with OpStatsLogger.registerSuccessfulEvent @zymap 
<img width="855" alt="image" src="https://user-images.githubusercontent.com/42990025/191929953-bdac2d25-3fbe-4697-b3e9-f4942ed7c074.png">

Reply:
I think we can both use unit.toMills

Reply:
> I think we can both use unit.toMills

goog idea, I will update it

Reply:
@Shoothzj have a look the new update for unit.toMillis